SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -18,6 +18,12 @@ component UnrealServerRPCEndpointLegacy {
     bool ready = 1;
     event UnrealRPCPayload server_to_client_rpc_event;
     event UnrealPackedRPCPayload packed_server_to_client_rpc;
+}
+
+component UnrealServerToServerRPCEndpointLegacy {
+    id = 9974;
+    // Set to true when authority is gained, indicating that RPCs can be received
+    bool ready = 1;
```

Comment:
I think ready was for client server RPCs. For server to server we can't really unset it properly when something transitions.

Reply:
sure, if it's not necessary then very happy to remove it from here.

Reply:
removed